tests/generate-test-data.ts

```diff
@@ -25,4 +25,15 @@ const report = {
   }
 
   await fs.writeFile(path.join("tests", "end-to-end", "report.json"), JSON.stringify(report, null, 2));
+
+
+
+  const colorTests = (await fs.readdir(path.join("src", "color-retriever", "test-data"))).filter((n) => !n.includes("."));
+
+  for (const t of colorTests) {
+    const imageData = await helpers.loadPng(path.join("src", "color-retriever", "test-data", t, "input.png"));
+    const output = jsQR(imageData.data, imageData.width, imageData.height, {retrieveColors: true});
+
+    await fs.writeFile(path.join("src", "color-retriever", "test-data", t, "output.json"), JSON.stringify(output.colors, null, 2), "utf8");
+  }
```

Comment:
I actually don't think we strictly need this, since I think the test cases are sort of...stand alone. Now that we've created them I would expect them to not break as we change the library? If they do we can re-visit needing to autogen them, but for now would be fine leaving them standalone?

Reply:
Seems useful to leave for adding more color tests but splitting it out makes sense. I'll make it a separate file in the color retrieving folder?